crates/astria-sequencer/src/bridge/bridge_sudo_change_action.rs

```diff
@@ -24,6 +24,22 @@ use crate::{
 
 #[async_trait::async_trait]
 impl ActionHandler for BridgeSudoChangeAction {
+    async fn check_stateless(&self) -> Result<()> {
+        crate::address::ensure_base_prefix(&self.bridge_address)
+            .context("bridge address is invalid")?;
```

Comment:
same as previous, this is specifically wrong base prefix

Reply:
Clarified the error in:  1634af